src/expression/values.ts

```diff
@@ -5,9 +5,10 @@ import {Formatted} from './types/formatted';
 import {Padding} from './types/padding';
 import {VariableAnchorOffsetCollection} from './types/variable_anchor_offset_collection';
 import {ResolvedImage} from './types/resolved_image';
-import {NullType, NumberType, StringType, BooleanType, ColorType, ObjectType, ValueType, CollatorType, FormattedType, ResolvedImageType, array, PaddingType, VariableAnchorOffsetCollectionType} from './types';
+import {NullType, NumberType, StringType, BooleanType, ColorType, ObjectType, ValueType, CollatorType, FormattedType, ResolvedImageType, array, PaddingType, VariableAnchorOffsetCollectionType, ProjectionDefinitionType} from './types';
 
 import type {Type} from './types';
+import {ProjectionDefinition} from './types/projection_definition';
```

Comment:
nit picking - move this import next to the other imports of color, resolvedImage etc...
Keep the `import type` to be separated and last.